pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -162,6 +162,18 @@ type NodeResourcesMostAllocatedArgs struct {
 	Resources []ResourceSpec
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourcesBalancedAllocationArgs holds arguments used to configure NodeResourcesBalancedAllocation plugin.
+type NodeResourcesBalancedAllocationArgs struct {
+	metav1.TypeMeta
+
+	// Resources to be considered when scoring.
+	// The default resource set includes "cpu" and "memory" with an equal weight.
+	// Weight should be always set to 1 per the nature of the scroing plugin.
+	Resources []ResourceSpec
```

Comment:
alternatively, create a type that doesn't have weight. If, in the future, we decide that weight makes sense, you can swap the types.
But for now, we save the validation code.

Reply:
Did you think about this?

Reply:
Great suggestion!

Reply:
Done